sema/commons/web/parse_headers.py

```
@@ -0,0 +1,35 @@
+            "mode should be one of 'content-type' (default) "
+            "or 'content-disposition'"
+        )
+    params = msg[mode].params
+    return main, params
+
+
+def get_parsed_header(
+    headers: dict[str, str],
+    key: str,
+) -> tuple[str, dict[str, str]]:
+    return parse_header(headers.get(key, None), key)
```

Ruff (0.8.2)

Use `headers.get(key)` instead of `headers.get(key, None)`

Replace `headers.get(key, None)` with `headers.get(key)`

(SIM910)